earhorn/main.py

```diff
@@ -127,6 +137,7 @@ def cli(
     def stop_handler(_signum, _frame):
         logger.info("stopping...")
         stop_event.set()
+        logger.info(get_files_and_upload("earhorn_segments.csv", archive_path))
```

Comment:
We cannot wait for the entire program to stop to finally upload the files, if you let the software run for 3 month you already have a huge amount of data to upload. In addition the local storage capacity might be limited, specially if one expect to store the files in s3.